exec/vm.go

```diff
@@ -272,7 +282,11 @@ func (vm *VM) ExecCode(fnIndex int64, args ...uint64) (rtrn interface{}, err err
 			if r := recover(); r != nil {
 				switch e := r.(type) {
 				case error:
-					err = e
+					if e.Error() == "return" {
```

Comment:
this is not sound and hard to understand, define a private struct: 
```golang
type normalReturn struct {}

const NORMAL_RETURN = normalReturn{}

// in Ret:
panic(NORMAL_RETURN)

// in recover:
if r := recover(); r != nil {
		switch e := r.(type) {
		case normalReturn:
			err = nil
		case error:
			....
		}
```

Reply:
from the original code logic, maybe keep this unchanged and just `panic(nil)` in Ret function should be ok.